# msg_FE01_parameter_1_t - enumeration
uint8 value

#Pre-Defined ENUM Constants
	uint8 READ_REQUEST        = 0
	uint8 READ_RESPONSE       = 1
	uint8 READ_BEGIN          = 2
	uint8 READ_READY          = 3
	uint8 READ_BLOCK          = 4
	uint8 READ_BLOCK_SUCCESS  = 5
	uint8 READ_BLOCK_FAIL     = 6
	uint8 READ_DONE           = 7
	uint8 READ_SUCCESS        = 8
	uint8 READ_FAILED         = 9
	uint8 WRITE_REQUEST       = 10
	uint8 WRITE_RESPONSE      = 11
	uint8 WRITE_BEGIN         = 12
	uint8 WRITE_READY         = 13
	uint8 WRITE_BLOCK         = 14
	uint8 WRITE_BLOCK_SUCCESS = 15
	uint8 WRITE_BLOCK_FAIL    = 16
	uint8 WRITE_DONE          = 17
	uint8 WRITE_SUCCESS       = 18
	uint8 WRITE_FAILED        = 19
